src/confcom/azext_confcom/data/rules.rego

```diff
@@ -526,6 +583,7 @@ allow_env(p_process, i_process, s_name) {
     print("allow_env: i env =", i_process.Env)
 
     every i_var in i_process.Env {
+        print("allow_env: i_var =", i_var)
```

Comment:
Do we want these prints?

Reply:
For this comment and the next one, the `rules.rego` is a file pulled from hcsshim releases. Updating it is just for transparency, so customers have a better understanding of what enforcement points are used.